lib/graphql/card.ts

```diff
@@ -214,6 +221,36 @@ export const SET_CARD_HOLDER_REPRESENTATION = `mutation setCardHolderRepresentat
   )
 }`;
 
+export const ADD_GOOGLE_PAY_CARD_TOKEN = `mutation(
+  $id: String!,
```

Comment:
Shall we make it consistent by having `,` for both lines or none of them (and in the delete mutation too)?

Yes I know we have this inconsistency everywhere 🙁